api/SVGComponentTransferFunctionElement.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGComponentTransferFunctionElement",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "5"
```

Comment:
This is one of the interfaces without instances in the tests mentioned in https://github.com/mdn/browser-compat-data/pull/7197#discussion_r516655475, but this data looks plenty plausible anyway, and based on https://caniuse.com/svg-filters it couldn't really have been earlier than this.